src/site/content/en/blog/lighthouse-ci/index.md

```diff
@@ -0,0 +1,351 @@
+</figure>
+
+1.  In the root of your repository, create a directory named `.github/workflows`. The [workflows](https://help.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow#about-workflows) for your project will go in this directory. A workflow is a process that runs at a predetermined time (for example, when code is pushed) and is composed of one or more actions.
+
+```shell
+mkdir .github
+mkdir .github/workflows
+```
+
+2.  In `.github/workflows` create a file named `lighthouse-ci.yaml`. This file will hold the configuration for a new workflow. 
+
+```shell
+touch lighthouse-ci.yaml
+```
+
+3.  Add the following text to `lighthouse-ci.yaml`. 
+
+```yaml
+name: Build project and run Lighthouse CI
+on: [push]
+jobs:
+  lhci:
+    name: Lighthouse CI
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Use Node.js 10.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 10.x
+      - name: npm install
+        run: |
+          npm install
+      - name: run Lighthouse CI
+        run: |
+          npm install -g @lhci/cli@0.3.x
+          lhci autorun --upload.target=temporary-public-storage || echo "LHCI failed!"
+```
+
+This configuration sets up a workflow consisting of a single job that will run whenever new code is pushed to the repository. This job has four steps:
+
+*   Check out the repository that Lighthouse CI will be run against
+*   Install and configure Node
+*   Install required npm packages
+*   Run Lighthouse CI and upload the results to temporary public storage.
+
+4.  Commit these changes and push them to GitHub. If you've correctly followed the steps above, pushing code to GitHub will trigger running the workflow you just added.
+
+5.  To confirm that Lighthouse CI has triggered and to view the report it generated, go to the "Actions" tab of your project. You should see the "Build project and Run Lighthouse CI" workflow listed under your most recent commit.
+
+<figure class="w-figure">
+  <img src="./github-action2.png" alt="Screenshot of the Github 'Settings' tab" class="w-screenshot">
+</figure>
+
+You can navigate to the Lighthouse report corresponding to a particular commit from the "Actions" tab. Click on the commit, click on the "Lighthouse CI" workflow step, then expand the results of the "run Lighthouse CI" step.
+
+<figure class="w-figure">
+  <img src="./github-action3.png" alt="Screenshot of the Github 'Settings' tab" class="w-screenshot">
+</figure>
+
+
+You've just set up a GitHub Action to run Lighthouse CI. This will be most useful when used in conjunction with a GitHub [status check](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-status-checks). 
+
+A status check, if configured, is a message that appears on every PR and typically includes information such as the results of a test or the success of a build.
```

Comment:
```suggestion
You've just set up a GitHub Action to run Lighthouse CI. This will be most useful when used in conjunction with a GitHub [status check](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-status-checks). 

### Set up a GitHub status check {: #github-status-checks }

A status check, if configured, is a message that appears on every PR and typically includes information such as the results of a test or the success of a build.
```